src/common/types/types.cpp

```diff
@@ -810,23 +810,30 @@ std::unique_ptr<LogicalType> LogicalTypeUtils::parseStructType(const std::string
         LogicalTypeID::STRUCT, std::make_unique<StructTypeInfo>(parseStructTypeInfo(trimmedStr)));
 }
 
+std::unique_ptr<LogicalType> MapType::createMapType(
+    std::unique_ptr<LogicalType> keyType, std::unique_ptr<LogicalType> valueType) {
+    std::vector<std::unique_ptr<StructField>> structFields;
+    structFields.emplace_back(
```

Comment:
Use push_back instead of emplace_back

Reply:
I'd just moved the code from `parseMapType`, but sure, this is a good opportunity to change that.